documentation/src/docs/recovery-procedures/table-segment-recovery.md

```diff
@@ -134,6 +134,13 @@ Also please note that the below described procedure assumes the use of "File Sys
       Ex:-
         data-recovery tableSegment-recovery /copied/chunk/directory completedTransactionsBatch-0 /output/chunk/directory
     ```
+ 
+### Note:        
+   Once we are done with the above step, we would have the recovered or corrected Attribute Index Segment Chunks created in `\output\chunk\directory`
+   A very important thing to do at this juncture would be to identify if we are dealing with a Metadata Segment. Please note that all Metadata Segments
```

Comment:
Consider rewriting/shortening this paragraph to something like:
`**Important**: If we are dealing with Metadata Table Segments, we can skip the steps that follow and directly continue with steps outline in [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/recovery-procedures/metadata-table-segment-recovery.md). Metadata Table Segments in Pravega either begin with metadata_<containerId> or storage_metadata_<containerId>`.

Please, try to keep technical text short and direct, otherwise we are just making things hard for the user.

Reply:
updated with above.